lib/internal/timers.js

```diff
@@ -590,12 +591,53 @@ function getTimerCallbacks(runNextTicks) {
   };
 }
 
+const Immediate = class Immediate {
```

Comment:
Why is the `const` assignment needed here?

Reply:
I have no idea lol... I copied this over as is from the existing definition in `lib/timers.js`